.github/workflows/main.yml

```diff
@@ -130,15 +130,15 @@ jobs:
               make -f Makefile test-clevis-loop
             toolchain: 1.58.1  # CURRENT DEVELOPMENT RUST TOOLCHAIN
             components: cargo
-            image: fedora:34  # CURRENT DEVELOPMENT ENVIRONMENT
+            image: fedora:35  # CURRENT DEVELOPMENT ENVIRONMENT
     runs-on: ubuntu-18.04
     container:
       image: ${{ matrix.image }}
       options: --privileged -v /dev:/dev
     services:
       tang:
         # yamllint disable rule:line-length
-        image: ghcr.io/stratis-storage/stratisd/tang:34  # CURRENT DEVELOPMENT ENVIRONMENT
+        image: docker.pkg.github.com/stratis-storage/stratisd/tang:35  # CURRENT DEVELOPMENT ENVIRONMENT
```

Comment:
^docker.pkg.github.com^ghcr.io

It's where GitHub is migrating all the images. There's two places you should make the change.

Reply:
OK; updated via force-push.